pro-planner-client/src/redux/pollSlice.js

```diff
@@ -4,7 +4,6 @@ const pollSlice = createSlice({
     name: 'poll',
     initialState: {
         polls: [],
```

Comment:
Out of curiosity, why did you go with an array for polls?

Reply:
I simply thought each plan can have many polls, so I just used an array. Do you have any suggestions that work better and are more intuitive? I'm not experienced with Redux, so it would be really nice if you could provide any advice regarding my code!

Reply:
I personally would've used an object to handle this, just to make searching for the right poll easier when doing modifications or deletion.

Reply:
You are right, it makes more sense with using object. I pushed another commit reflecting your suggestion!